code/game/atoms.dm

```diff
@@ -168,8 +171,18 @@ directive is properly returned.
 	if(loc)
 		return loc.return_gas()
 
+//returns how dissolveable this atom is, lower number means it takes longer to dissolve it
+/atom/proc/dissolvability(acid_strength)
+	return 1
 
+//returns how long it takes to apply acid on this atom
```

Comment:
/// not // 4head

Also we use the dissolvability to determine if something can be melted at all, so the autodoc isn't quite accurate.